### PR TITLE
omegat: init at 4.1.5.2

### PR DESCRIPTION
**Pull request by t184256**
OmegaT is a Computer-Assisted Translation tool written in Java.
This derivation doesn't build it from source, but rather downloads
a 'Cross-platform without JRE' zip file.
I took the liberty of shorting the upstream version naming of
"4.1.5 update 2 Beta" to a more concise "4.1.5.2".

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/omegat.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  unpackCmd = "unzip -o $curSrc";  # tries to go interactive without -o
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r lib docs images plugins scripts *.txt *.html OmegaT.jar $out/
+
+    cat > $out/bin/omegat <<EOF
+    #! $SHELL -e
+    CLASSPATH="$out/lib"
+    exec ${jdk}/bin/java -jar -Xmx1024M $out/OmegaT.jar "$@"
```
Comment (Infinisil) [inline pkgs/applications/misc/omegat.nix]: The `"$@"` needs to be escaped, because like this it will be the args from the package builder. Use `\"\$@\"`

**Review by infinisil — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/omegat.nix (hunk 1 above)
Comment (t184256) [reply pkgs/applications/misc/omegat.nix]: Hm, interesting catch. Changed and force-pushed.

File: pkgs/applications/misc/omegat.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  unpackCmd = "unzip -o $curSrc";  # tries to go interactive without -o
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r lib docs images plugins scripts *.txt *.html OmegaT.jar $out/
+
+    cat > $out/bin/omegat <<EOF
+    #! $SHELL -e
+    CLASSPATH="$out/lib"
+    exec ${jdk}/bin/java -jar -Xmx1024M $out/OmegaT.jar \"\$@\"
```
Comment (Infinisil) [inline pkgs/applications/misc/omegat.nix]: Oh sorry, my mistake, it only needs `"\$@"` to be escaped properly, the `"` escaping is taken care of by the heredoc

**Comment by t184256** [comment]
Why is it still marked as 'requested changes'?

**Review by infinisil — APPROVED** [review]
These reviews are done manually, I just didn't look at it again

Looks good now :)

**Pull request merged**